dex/src/main/scala/com/wavesplatform/dex/AddressWsMutableState.scala

```diff
@@ -15,7 +15,7 @@ case class AddressWsMutableState(address: Address,
                                  pendingSubscription: Set[ActorRef[WsAddressState]],
                                  changedSpendableAssets: Set[Asset],
                                  changedReservableAssets: Set[Asset],
-                                 ordersChanges: Map[Order.Id, WsOrder]) {
+                                 ordersChanges: Map[Order.Id, WsOrder]) { // TODO Probably use an ordered Map and pass it to WsAddressState
```

Comment:
How will we sort the keys? Or you mean `LinkedHashMap`?

Reply:
Probably, there is TreeMap also and other alternatives :)

Reply:
I just wonder what the way of sorting `Order.Ids` do you suggest
Bytes comparison?

Reply:
Yep